pyproject.toml

```diff
@@ -179,6 +180,10 @@ select = [
 ]
 ignore = [
     "ISC001",   # flake8-implicit-str-concat: single-line-implicit-string-concatenation  # incompatible with the formatter
+    "S105",     # flake8-bandit: hardcoded password
+    "S106",     # flake8-bandit: hardcoded password
+    "S603",     # flake8-bandit: subprocess-without-shell-equals-true
+    "S320",     # flake8-bandit: xml parsing vulnerable to XML attacks
```

Comment:
In our codebase, we often parse XML input with lxml, like e.g. `etree.parse(file)`. lxml is considered unsafe by ruff. But I think in our case it's okay.